.github/mergify.yml

```
@@ -0,0 +1,18 @@
+queue_rules:
+  - name: default
+    conditions:
+      - base=master
+
+pull_request_rules:
+  - name: automerge to master with label automerge and branch protection passing
+    conditions:
+      - "#approved-reviews-by>1"
```

We can change this to be 
```suggestion
      - "#approved-reviews-by>0"
```
So even a single review can merge it as we don't have enough resources actively working on this to always require 2 reviews

---

Agreed.